Makefile.dep

```diff
@@ -606,6 +606,14 @@ ifneq (,$(filter random,$(USEMODULE)))
   endif
 endif
 
+ifneq (,$(filter random_seed,$(USEMODULE)))
+  USEMODULE += luid
+  USEMODULE += xtimer
+
+  FEATURES_REQUIRED += periph_hwrng
+  FEATURES_REQUIRED += periph_adc
```

Comment:
But I think we desperately need to rethink the dependency system in the near future, this is getting annoying...